drivers/led_strip/led_strip_fake.c

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2025 Gielen De Maesschalck <gielen.de.maesschalck@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/device.h>
+#include <zephyr/drivers/led_strip.h>
+#include <zephyr/drivers/led_strip/led_strip_fake.h>
+#include <zephyr/fff.h>
+
+#ifdef CONFIG_ZTEST
+#include <zephyr/ztest.h>
+#endif /* CONFIG_ZTEST */
```

Comment:
```suggestion
#include <zephyr/ztest.h>
```

You must enable `CONFIG_ZTEST` to use it.